ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml

```diff
@@ -127,6 +127,65 @@ presubmits:
         secret:
           secretName: openshift-merge-bot
     trigger: (?m)^/test( | .* )check-gh-automation-tide,?($|\s.*)
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - ^master$
+    - ^master-
+    cluster: build03
+    context: ci/prow/image-mirroring-config-validation
+    decorate: true
+    labels:
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-release-ci-image-mirroring-config-validation
+    rerun_command: /test image-mirroring-config-validation
+    run_if_changed: ^core-services/image-mirroring/_config.yaml
+    spec:
+      containers:
+      - args:
+        - --release-repo-git-sync-path=/var/repo/release
+        - --registry-config=/etc/push/.dockerconfigjson
+        - --config=/var/repo/release/core-services/image-mirroring/_config.yaml
+        - --validate-config-only=true
+        image: ci-images-mirror
+        name: ci-images-mirror
+        resources:
+          requests:
+            cpu: 500m
+            memory: 2Gi
+        volumeMounts:
+        - mountPath: /etc/push
+          name: push-credentials
+          readOnly: true
+        - mountPath: /var/repo
+          name: release
+      - args:
+        - --repo=https://github.com/openshift/release.git
+        - --branch=master
+        - --wait=30
+        - --root=/tmp/git-sync
+        - --max-sync-failures=3
+        command:
+        - /git-sync
+        env:
+        - name: GIT_SYNC_DEST
+          value: release
+        image: quay-proxy.ci.openshift.org/openshift/ci:ci_git-sync_v4.2.3
+        name: git-sync
+        resources:
+          requests:
+            cpu: 500m
+            memory: 1Gi
+        volumeMounts:
+        - mountPath: /tmp/git-sync
+          name: release
+      volumes:
+      - name: push-credentials
+        secret:
+          secretName: registry-push-credentials-ci-images-mirror
```

Comment:
Let us try [this](https://github.com/openshift/release/blob/72156ca5c3cc20f40af8029c0b6c29a03b6edd49/core-services/ci-secret-bootstrap/_config.yaml#L3668) one:

Pros:
* I cannot think of write permissions to any registry is needed in the validation mode.
* It can thus be run on any cluster: `registry-pull-credentials` is available everywhere while `registry-push-credentials-ci-images-mirror` is only on `app.ci`.

Cons:
* Rehearsal (the validation mode) is done in a different environment from the real one. But I think I am fine with it, at least for the first iteration.

Reply:
In general, you cannot really rehearse it until https://github.com/openshift/ci-tools/pull/4245 gets merged.